Sources/BraintreePaymentFlow/BTLocalPaymentResult.swift

```diff
@@ -0,0 +1,58 @@
+@objcMembers public class BTLocalPaymentResult: BTPaymentFlowResult {
```

Comment:
I am not 100% sure when we need to import @BraintreeCore or not.

Reply:
You import a module when you need to access it's classes & methods. In this case, `BTPostalAddress` is imported in parent class `BTPaymentFlowResult`.

The `canImport` syntax is for building our Demo app, and also for SPM _I believe_